redaxo/src/core/lib/console/user/create.php

```diff
@@ -0,0 +1,82 @@
+<?php
+
+use Symfony\Component\Console\Exception\InvalidArgumentException;
+use Symfony\Component\Console\Input\InputArgument;
+use Symfony\Component\Console\Input\InputOption;
+use Symfony\Component\Console\Input\InputInterface;
+use Symfony\Component\Console\Output\OutputInterface;
+
+/**
+ * @package redaxo\core
+ *
+ * @internal
+ */
+class rex_command_user_create extends rex_console_command
+{
+    protected function configure()
+    {
+        $this
+            ->setDescription('Create a new user')
+            ->addArgument('login', InputArgument::REQUIRED, 'Login')
+            ->addArgument('password', InputArgument::OPTIONAL, 'Password')
+            ->addOption('username', null, InputOption::VALUE_OPTIONAL, 'Username')
+            ->addOption('is_admin', null, InputOption::VALUE_OPTIONAL, 'Grant admin permissions', false)
```

Comment:
Glaube Unterstriche in Optionsnamen sind eher ungewöhnlich. Aktuell sähe das ja so aus: `--is_admin=1`.
Wir wäre es einfach mit `--admin` ohne Value?